src/client/lobby/room-list.spec.js

```diff
@@ -14,8 +14,8 @@ describe('<RoomList />', () => {
 
     it('renders the list of open rooms correctly', () => {
         const wrapper = shallow(<RoomList roomList={mockList} />);
-        wrapper.find('p').forEach((current, index) => {
-            expect(current.text()).toEqual(mockList[index]['name']);
+        wrapper.find('button').forEach((current, index) => {
+            expect(current.text()).toEqual(mockList[index]['name']+mockList[index]['users'].length);
```

Comment:
Use ES6 `${}` for string concatenation.
Also better to have `mockList[index].name` and `mockList[index].users`